content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -195,9 +188,9 @@ custom:
 ...
```

Comment:
We assume here is more config without giving any explanation - it might be a bit confusing for beginners.

Maybe we could also link to the actual source in the sample project here? Currently the sample is only outlined at the very end in the conclusion.

Reply:
Fixed this. Can you check again?